pkg/operation/botanist/logging.go

```diff
@@ -62,28 +57,52 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 		return err
 	}
 
+	if b.isShootEventLoggerEnabled() {
+		if err := b.Shoot.Components.Logging.ShootEventLogger.Deploy(ctx); err != nil {
+			return err
+		}
+	} else {
+		if err := b.Shoot.Components.Logging.ShootEventLogger.Destroy(ctx); err != nil {
+			return err
+		}
+	}
+
 	lokiValues := map[string]interface{}{
 		"global":   seedImages,
 		"replicas": b.Shoot.GetReplicas(1),
 	}
 
+	// check if loki is enabled in gardenlet config, default is true
+	if !gardenlethelper.IsLokiEnabled(b.Config) {
+		// because ShzotNodeLogging is installed a part like Loki sidecar container
```

Comment:
```suggestion
		// because ShootNodeLogging is installed a part like Loki sidecar container
``` 

I also don't understand what is meant by ` is installed a part like Loki sidecar container`

Reply:
I have to stop coding when I am drunk :(.
```text
// Because ShootNodeLogging is installed as part of the Loki pod
// we have to delete it too in the case previously was deployed
```